<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>10</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>실행</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_img_dst">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_file_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>10</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>이미지 로드</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_img_src">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>원본 이미지</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>570</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>결과 이미지</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_threshold">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>THRESH_BINARY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_BINARY_INV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_TRUNC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_TOZERO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_TOZERO_INV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_MASK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_OTSU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THRESH_TRIANGLE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ADAPTIVE_THRESH_MEAN_C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ADAPTIVE_THRESH_GAUSSIAN_C</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>이진화 옵션</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_thr_value">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>600</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="slider_thr">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>640</y>
      <width>851</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>127</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_thr">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>640</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>127</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
